README.md

```diff
@@ -206,7 +206,7 @@ This feedback is important for us to deliver a useful product.
 
 [HPE OneView Firmware Management White Paper](http://hpe.com/info/OneView/docs)
 
-Note: Currently this SDK supports OneView API 3000 minimally where we can test OneView API 3000 version with this SDK. No new fields have been added/deleted to support API3000 version. Complete support will be done in next releases.If  API version is not provided then appliance's API version will be used. If API version used is not supported then error will be thrown.
+Note: Currently this SDK supports OneView API 3200 minimally where we can test OneView API 3200 version with this SDK. No new fields have been added/deleted to support API3000 version. Complete support will be done in next releases.If  API version is not provided then appliance's API version will be used. If API version used is not supported then error will be thrown.
```

Comment:
change this "No new fields have been added/deleted to support API3000 version" to "No new fields have been added/deleted to support API 3200 version"